doc/qbk/3.4.path.qbk

```diff
@@ -147,30 +147,80 @@ clearly not intended. This table shows a URL string, a
 modification operation, and the URL string which results from
 applying the operation:
 
+[/
```

Comment:
Just a comment?

Reply:
Yes its just a note to help us think about ways to break The Algorithm